module3/homework/inner_product/arithmeticAverage.cpp

```diff
@@ -0,0 +1,13 @@
+#include <cmath>
+#include <functional>
+#include <numeric>
+#include <vector>
```

Comment:
Do we need to include <vector> if we have already included it in the hpp file?

Reply:
Hmm, afaik include what you use? In this case vector operators.

Reply:
Oki thanks <3 I didn't know about this rule.